ideconfigs/intellij-complex-nextraq-style.xml.xml

```diff
@@ -0,0 +1,608 @@
+<code_scheme name="NexTraqStyle" version="173">
+  <option name="OTHER_INDENT_OPTIONS">
+    <value>
+      <option name="CONTINUATION_INDENT_SIZE" value="4" />
+    </value>
+  </option>
+  <option name="ENABLE_SECOND_REFORMAT" value="true" />
+  <option name="KEEP_CONTROL_STATEMENT_IN_ONE_LINE" value="false" />
```

Comment:
![image](https://user-images.githubusercontent.com/61692292/234630816-8b84cdc6-ec2a-4945-881c-148a1a9f5997.png)

Reply:
heh...I didn't even know this was here.  Generally, I have almost all inspections turned on in IntelliJ.  However, there are a few that conflict with each other, and a choice has to be made.  For example, there are two inspections for requiring local variables to be final or not final.  Since there is also an inspection that warns of variable reassignment (and one for parameter variable reuse), I choose the one that warns when local variables are final since it seems like boilerplate noise.